details/operator-sdk/controllers/astraconnector_controller.go

```diff
@@ -151,7 +155,7 @@ func (r *AstraConnectorController) Reconcile(ctx context.Context, req ctrl.Reque
 
 	// POST/PUT Managed Cluster
 	_ = r.updateAstraConnectorStatus(ctx, astraConnector, natsSyncClientStatus)
-	return ctrl.Result{}, nil
+	return ctrl.Result{RequeueAfter: time.Minute * conf.Config.ErrorTimeout()}, nil
```

Comment:
This is when the reconciliation ends without any issues, should be no requeue @colep-ntap